test/e2e/specs/site-editor/zoom-out.spec.js

```diff
@@ -27,8 +27,8 @@ test.describe( 'Zoom Out', () => {
 
 		// Check that the html is scaled.
 		await expect( html ).toHaveCSS(
-			'transform',
-			'matrix(0.67, 0, 0, 0.67, 0, 0)'
+			'scale',
+			new RegExp( /0\.[5-8][0-9]*/, 'i' )
```

Comment:
Why `0.5` to `0.89`?

Reply:
[toHaveCSS](https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-have-css) only accepts string or regex. And I did a range so it would allow us to update the the zoom level scaling or sizing without this failing. .5-0.89 seemed like a reasonable range.